Joystream-Role-Bot/src/controls/control.ts

```diff
@@ -150,13 +155,23 @@ export const setMemberRole = async (client: Client): Promise<void> => {
 
       const roleUpdatePromise =
         memberRole.status.__typename === "WorkerStatusActive"
-          ? discordMember.roles.add(role)
-          : discordMember.roles.remove(role);
+          ? await discordMember.roles.add(role)
+          : await discordMember.roles.remove(role);
```

Comment:
Those `await`s are not needed since the promise is awaited in the line below (`await roleUpdatePromise;`)